docs/rofl/features/proxy.mdx

```diff
@@ -0,0 +1,88 @@
+# ROFL Proxy
+
+The ROFL proxy automatically generates public HTTPS URLs for services in your
+ROFL app. Simply publish a port in your `compose.yaml` and the proxy handles
+TLS certificates and routing.
+
+## Enabling the Proxy
+
+To expose a port from your container, simply publish it in
+your `compose.yaml` file:
+
+```yaml title="compose.yaml"
+services:
+  frontend:
+    image: docker.io/hashicorp/http-echo:latest
+    ports:
+      - "5678:5678" # Expose container port 5678 on host port 5678
+```
+
+After deploying your ROFL app, you can find the generated URL by
+running `oasis rofl machine show`:
+
+```shell
+oasis rofl machine show
+```
+
+The output will contain a `Proxy` section with the URL for each published port:
+
+```
+Proxy:
+  Domain: m602.test-proxy-b.rofl.app
+  Ports from compose file:
+    5678 (frontend): https://p5678.m602.test-proxy-b.rofl.app
+```
+
+:::info Prerequisites
+To use the ROFL proxy:
```

Comment:
Can we move this to some general section on how to upgrade to the latest version of the ROFL app artifacts and then just link it from here? This is generally useful as people should be upgrading to get the latest improvements.

Reply:
I would just remove the prerequisites from this chapter to avoid maintenance.

Reply:
agree that it's better not to have it in this chapter, since we added the same info in the troubleshooting doc